packages/selected-network-controller/CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [13.0.0]
+
+### Changed
+
+- `getProviderAndBlockTracker` now returns the `NetworkController`'s globally selected network client proxy if the `domain` arg is either `metamask` or a snap (identified as starting with `npm:` or `local:`) ([#4259](https://github.com/MetaMask/core/pull/4259))
+- **BREAKING:** Now when `setNetworkClientIdForDomain` is called with a snap's domain (identified as starting with `npm:` or `local:`), the `domain` will not be added to state and no proxy will be created for this domain in the `domainProxyMap` ([#4258](https://github.com/MetaMask/core/pull/4258))
+  - In order to remove snaps that made it into `domains` state prior to this change, consumers will need to run a migration.
+- Bump `@metamask/json-rpc-engine` to `^8.0.2` ([#4234](https://github.com/MetaMask/core/pull/4234))
+- Bump `@metamask/base-controller` to  `^5.0.2` ([#4232](https://github.com/MetaMask/core/pull/4232))
```

Comment:
Nit: Missing newline after this (probably because I wrote that suggestion before you fixed the extra newline lint error)

```suggestion
- Bump `@metamask/base-controller` to  `^5.0.2` ([#4232](https://github.com/MetaMask/core/pull/4232))

```